src/odemis/odemisd/main.py

```diff
@@ -288,7 +288,7 @@ def _instantiate_component(self, name):
                 pass
             raise ValueError("Failed to instantiate component %s" % name)
         else:
-            children = self._instantiator.get_children(comp)
```

Comment:
Technically, here, we only care about the children anyway. So we don't need to get also the dependencies, and then filter them with the "delegated_children". => just call `get_children()` and add them to the alive.

BTW, you could probably rename `get_delegated_children()` to something without "delegated"... maybe `get_children_names()`.